apps/condo/domains/property/utils/clientSchema/Property.ts

```diff
@@ -17,6 +17,9 @@ import { BuildingMap } from '@condo/domains/property/components/panels/Builder/M
 export interface IPropertyUIState extends Property {
     id: string
     address: string
+    addressMeta: {
+        data: any
```

Comment:
```suggestion
        data: AddressMeta
```

Reply:
Data is not of type of AdressMeta, it's just field of it....So I guess it should be `addressMeta: AddressMeta`
You can find it in `@condo/domains/common/utils/addressApi/AddressMeta`